aws-s3-transfer-manager/src/operation/upload/builders.rs

```diff
@@ -917,6 +917,6 @@ impl crate::operation::upload::input::UploadInputBuilder {
     ) -> Result<UploadHandle, crate::error::Error> {
         let mut fluent_builder = client.upload();
         fluent_builder.inner = self;
-        fluent_builder.send().await
+        fluent_builder.initiate()
```

Comment:
if we're changing the name-and-asyncness of `UploadFluentBuilder::send()` then we should change the name-and-asyncness of `UploadInputBuilder::send_with()`

Reply:
Thanks, yeah, I can make it consistent once we have agreed on the `initiate` name in the previous feedback.